docs/content/v0.20/contributor.md

```diff
@@ -0,0 +1,60 @@
+# KubeStellar Contributors
+
+**NOTE**: work in progress... write more
+
+## Check key pre-requisites for KubeStellar
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script offers a convenient way to check key pre-requisites needed for KubeStellar deployment and use case scenarios.

```

Comment:
"key" is too vague. Point to the existing pre-reqs.md and explain that the check_pre_req.sh script checks all categories in that doc and more.